File: Reserva-Hotel-com-Front-main/ProjetoCs-Reserva_de_Hotel-main/ReservaDeHotel/Models/Avaliacao.cs
using System.ComponentModel.DataAnnotations;

namespace ReservaDeHotel.Models;
public class Avaliacao
{
    [Key]
    public int? IdAvaliacao {get;set;}
    public int? HotelId { get; set;}
    public List<Hotel>? Hoteis { get; set;}
    public int? AvaliacaoEstrelas { get; set;}
    public string? Comentario { get; set;}
    public DateTime? DataAvaliacao { get; set;}
}

File: Reserva-Hotel-com-Front-main/ReservaHotelFront/src/app/Avaliacao.ts
import { Hotel } from './Hotel';

export class Avaliacao {
    idAvaliacao: number = 0;
    avaliacaoEstrelas: number = 0;
    comentario: string = '';
    dataAvaliacao: Date = new Date();
    hotelId: number = 0;
    hotel: Hotel | undefined;
}
